dynamic-plugins.default.yaml

```diff
@@ -518,6 +518,44 @@ plugins:
               clientId: "${KEYCLOAK_CLIENT_ID}"
               clientSecret: "${KEYCLOAK_CLIENT_SECRET}"
 
+  - package: ./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-bitbucket-cloud
+    disabled: true
+    pluginConfig:
+      # https://backstage.io/docs/integrations/bitbucketCloud/discovery#configuration
+      catalog:
+        providers:
+          bitbucketCloud:
+            yourProviderId: # identifies your ingested dataset
+              catalogPath: /catalog-info.yaml # default value
```

Comment:
Can we use env variables here and below, in order to have this default config be something meaninful and valid (provided we defined the expected env variables).
This is how it has been until now for other plugins, and also historically for the showcase in general in the `app-config.yaml`.

Reply:
This was the config I copied from the backstage documentation.

Reply:
I get it, but the config fragments in this file  are expected to provide default configs that work.

Reply:
Don't we want to fix this ?